front/app/containers/App/index.tsx

```diff
@@ -208,16 +208,21 @@ const App = ({ children }: Props) => {
     const handleCustomRedirect = () => {
       const { pathname } = location;
       const urlSegments = pathname.replace(/^\/+/g, '').split('/');
+      const pathnameWithoutLocale = removeLocale(pathname).pathname?.replace(
+        /\//,
+        ''
+      );
+
       if (
         appConfiguration &&
         appConfiguration.data.attributes.settings.redirects
       ) {
         const { rules } = appConfiguration.data.attributes.settings.redirects;
         rules.forEach((rule) => {
           if (
-            urlSegments.length === 2 &&
+            urlSegments.length > 1 &&
             includes(locales, urlSegments[0]) &&
-            urlSegments[1] === rule.path
+            pathnameWithoutLocale === rule.path
           ) {
             window.location.href = rule.target;
           }
```

Comment:
Good that you fixed, but given there have not been complaints I wonder if this is used. Maybe we can delete it altogether if it's not?

Reply:
It is used - [we checked](https://citizenlab.atlassian.net/browse/CL-2917) :))